tests/checker/bad/message_field.scilla

```diff
@@ -0,0 +1,42 @@
+scilla_version 0
+
+library Msg_Field_Test
+
+let one_msg = 
+  fun (msg : Message) => 
+    let nil_msg = Nil {Message} in
+    Cons {Message} msg nil_msg
+
+contract Msg_Field_Test ()
+
+(* The type of arguments to fields in Message is incorrect *)
+transition malMessage3(num1 : Int32)
+  (* duplicate field *)
```

Comment:
Copy-paste error, I think. All the transitions have this comment, yet none of the messages have duplicate fields.